src/swift_server.rs

```diff
@@ -363,10 +366,53 @@ pub async fn start_server() {
         listener_metrics.local_addr().unwrap()
     );
 
-    let _ = tokio::try_join!(
-        axum::serve(listener, app),
-        axum::serve(listener_metrics, metrics_app)
-    );
+    // RPC sim loop to avoid rpc cold starts when orders are infrequent
+    // Build tx once and just resign with new blockhash
+    let rpc_sim_loop = tokio::spawn(async {
+        let sender = Keypair::new();
+        let receiver = Keypair::new();
+        let instruction = solana_sdk::system_instruction::transfer(
+            &sender.pubkey(),
+            &receiver.pubkey(),
+            1_000_000_000u64,
+        );
+
+        let mut interval = tokio::time::interval(Duration::from_secs(5));
+
+        loop {
+            interval.tick().await;
+            let message = Message::try_compile(
+                &sender.pubkey(),
+                &[instruction.clone()],
+                &[],
+                Hash::default(),
+            )
+            .unwrap();
+            let versioned_message = VersionedMessage::V0(message);
```

Comment:
just a nit: but maybe can build once and `clone` the `versioned_message` each loop